unstructured/partition/email.py

```diff
@@ -258,13 +251,20 @@ def partition_email(
         #    <li>Item 1</li>=
         #    <li>Item 2<li>=
         # </ul>
+        if not encoding:
+            encoding = "utf-8"
         list_content = content.split("=\n")
         content = "".join(list_content)
         elements = partition_html(text=content, include_metadata=False)
         for element in elements:
             if isinstance(element, Text):
                 _replace_mime_encodings = partial(replace_mime_encodings, encoding=encoding)
-                element.apply(_replace_mime_encodings)
+                try:
+                    element.apply(_replace_mime_encodings)
+                except UnicodeDecodeError:
+                    # If decoding fails, try decoding with default encoding (utf-8)
+                    _replace_mime_encodings = partial(replace_mime_encodings)
```

Comment:
missing arg in the partial?

Reply:
If decoding fails, it will try decoding with the default value (UTF-8) of the `encoding` parameter of the `replace_mime_encodings` function. I will fix the missing arg in the partial.

https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/cleaners/core.py#L190